DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/CourseSetupController.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.CourseSetup
+{
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Authorize(Policy = CustomPolicies.UserCentreAdmin)]
+    [Route("/TrackingSystem/CourseSetup")]

```

Comment:
This route is different to the one on the ticket?

Reply:
It is. After a little discussion (can be found in the comments of the HLD), the folder setup was preferred to be TrackingSystem/CourseSetup. The original TrackingSystem/CentreCourseSetup is still feasible, but I thought it would be best to keep it consistent with the folders, and avoid confusion with the TrackingSystem/Centre/... pages.